### ISSUE TITLE
Migrating from v4.0.0-beta.24 to v4.0.6

### ISSUE THREAD

**Issue by pottercomuneo** [aksonov/react-native-router-flux#3584]
### Version
Tell us which versions you are using:

- react-native-router-flux v4.0.6
- react v16.9.0
- react-native v0.61.2

-------
Hello everybody, after upgrading to v4.0.6 from v4.0.0-beta.24 I'm facing strange issues, like the back button not working or properties not correctly passed when pushing at new scene.
Does anyone know the main things to care about in this migration? Or is there anywhere to check the differences?
Thank you very much!

**Comment by aksonov** [comment]
Please use latest 4.1.x version instead

**Comment by pottercomuneo** [comment]
Hi Pavel! But isn't 4.1.x still in beta?
Thank you

**Comment by aksonov** [comment]
Yes, it's beta but it works well and it is based on latest react navigation 4.x and we use it in our production app
